public/js/agora-broadcast-client.js

```diff
@@ -131,21 +131,25 @@ async function createCameraStream(uid, deviceIds) {
       jQuery("#exit-btn").prop("disabled", false);
     }
     
-    window.agoraClient.publish(localStream, function (err) {
-      err && AgoraRTC.Logger.error('[ERROR] : publish local stream error: ' + err);
-    });
+    // window.agoraClient.publish(localStream, function (err) {
+    //   err && AgoraRTC.Logger.error('[ERROR] : publish local stream error: ' + err);
+    // });
+
+    if(!window.pre_call_device_test_enabled){
+      publishLocalStream(localStream);
+    }
 
     window.localStreams.camera.stream = localStream; // keep track of the camera stream for later
 
     /* Mute Audios and Videos Based on Mute All Users Settings */
-    if(window.mute_all_users_audio_video){
-      if(localStream.getVideoTrack() && localStream.getVideoTrack().enabled){
-        jQuery("#video-btn").trigger('click');
-      }
-      if(localStream.getAudioTrack() && localStream.getAudioTrack().enabled){
-        jQuery("#mic-btn").trigger('click');
-      }
-    }
+    // if(window.mute_all_users_audio_video){
```

Comment:
why has this been commented out?

Reply:
> why has this been commented out?

This should be work after stream is published (not in pre-call device test, so moved this code in publish stream function.), was testing this condition, will remove it in next PR (from commented section).

Reply:
> > why has this been commented out?
> 
> This should be work after stream is published (not in pre-call device test, so moved this code in publish stream function.), was testing this condition, will remove it in next PR (from commented section).

Fixed. Kindly check it.